frontend/demo/fusion/authentication/handle-session-expiration/login-view.ts

```diff
@@ -0,0 +1,53 @@
+import { customElement, html, LitElement, property } from 'lit-element';
+import { Router } from '@vaadin/router';
+import { LoginResult, login } from '@vaadin/flow-frontend';
+
+@customElement('login-view')
+export class LoginView extends LitElement {
+  @property({ type: Boolean })
+  private error = false;
+
+  private returnUrl = '/';
+
+  private onSuccess = (_: LoginResult) => { Router.go(this.returnUrl) };
+
+  render() {
+    return html`
+      <vaadin-login-overlay opened .error=${this.error} @login="${this.login}">
```

Comment:
```suggestion
      <vaadin-login-overlay opened .error="${this.error}" @login="${this.login}">
```

Consistency with quotes.

Reply:
done